Libraries/Utilities/Platform.android.js

```diff
@@ -22,7 +22,7 @@ const Platform = {
     const constants = require('NativeModules').AndroidConstants;
     return constants && constants.isTesting;
   },
-  select: (obj: Object) => obj.android,
+  select: (obj: Object) => 'android' in obj ? obj.android : obj.default,
```

Comment:
> Any difference over just obj.android || obj.default ?

`obj.android || obj.default` will ignore `falsy` values like `0`, `false`, `null` etc.

> Also, shouldn't we inline that in babel plugin instead of that condition here?

It should be done in both places